docs/guide/yaml/example_yaml/brooklyn-elasticsearch-catalog.bom

```diff
@@ -0,0 +1,126 @@
+brooklyn.catalog:
+  id: elasticsearch
+  version: 1.0
+  iconUrl: https://avatars0.githubusercontent.com/u/6764390?v=3&s=400
+  name: Elasticsearch
+  license: Apache-2.0
+  maintainer_name: Graeme Miller
+  maintainer_email: graeme.miller@cloudsoftcorp.com
```

Comment:
Would prefer to exclude maintainer name/email for the same reason as `@author` tags are discouraged.